libcxx/include/__memory/uninitialized_algorithms.h

```diff
@@ -646,6 +647,25 @@ __uninitialized_allocator_move_if_noexcept(_Alloc&, _Iter1 __first1, _Iter1 __la
 }
```

Comment:
Can we simplify the implementation of `__uninitialized_allocator_move_if_noexcept` now that it's only used in `__uninitialized_allocator_relocate`? Can we remove it and instead implement only `_relocate`?